app/todo/api.py

```diff
@@ -0,0 +1,37 @@
+from fastapi import APIRouter, Depends
+
+from app.depends.auth import get_current_user
+from app.depends.service import get_todo_service
+from app.todo.models import (DeleteTodo, DeleteTodoInfo,
+                             GetTodoInfo, GetTodoInfoList, InsertTodoInfo)
+from app.todo.service import BaseTodoService
+
+app_router = APIRouter()
+
+
+@app_router.get("/todoinfo", response_model=GetTodoInfoList)

```

Comment:
todo를 List view로 fetch하는 로직이므로 단순하게 `GET /todo`로 정의해도 좋을 것 같습니다! (p3)
```suggestion
@app_router.get("/todo", response_model=GetTodoInfoList)

```

Reply:
변경완료했습니다!
10fcdca476d40ccf1fd8c9d6a29a30c9f744a525